response_selectors/emotion_ranking_based_response_selector/server.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+
+import logging
+import numpy as np
+import requests
+import time
+from copy import deepcopy
+from os import getenv
+
+import sentry_sdk
+from flask import Flask, request, jsonify
+from common.utils import is_toxic_or_badlisted_utterance
+
+
+sentry_sdk.init(getenv("SENTRY_DSN"))
+
+logging.basicConfig(format="%(asctime)s - %(name)s - %(levelname)s - %(message)s", level=logging.INFO)
+logger = logging.getLogger(__name__)
+
+app = Flask(__name__)
+
+SENTENCE_RANKER_ANNOTATION_NAME = getenv("SENTENCE_RANKER_ANNOTATION_NAME")
+SENTENCE_RANKER_SERVICE_URL = getenv("SENTENCE_RANKER_SERVICE_URL")
+SENTENCE_RANKER_TIMEOUT = int(getenv("SENTENCE_RANKER_TIMEOUT"))
```

Comment:
```suggestion
SENTENCE_RANKER_TIMEOUT = float(getenv("SENTENCE_RANKER_TIMEOUT"))
```

Reply:
also compare file with the latest version of response selector original (which it is copied from) because I think there were some bug fixes, i want you to check whether you do not have same bugs here

Reply:
fixed. Also, added new test files as in [ranking_based_response_selector](https://github.com/deeppavlov/dream/tree/dev/response_selectors/ranking_based_response_selector): [test.py](https://github.com/deeppavlov/dream/blob/dev/response_selectors/ranking_based_response_selector/test.py) and [conftest.py](https://github.com/deeppavlov/dream/blob/dev/response_selectors/ranking_based_response_selector/conftest.py)